assets/app/view/create_game.rb

```diff
@@ -545,5 +520,146 @@ def update_inputs(title_change: false)
       store(:optional_rules, @optional_rules, skip: true)
       store(:visible_optional_rules, visible_rules)
     end
+
+    def select_game(meta)
+      @selected_game = meta
+
+      update_player_range(meta)
+
+      uncheck_game_variant
+      @selected_variant = nil
+      @game_variants = {}
+
+      uncheck_optional_rules
+      @optional_rules = []
+
+      preselect_variant(@selected_game)
+      preselect_optional_rules(@selected_game)
+
+      update_inputs(title_change: true)
+
+      `window.scroll(0, 0)`
+    end
+
+    def render_random_seed
+      h(:div, [
+          render_input(
+            'Random Seed',
+            id: :seed,
+            placeholder: 'Optional random seed',
+            label_style: { display: 'none' },
+            on: {
+              keyup: lambda { |e|
+                Native(e)['key'] == 'Enter' && e.JS.stopPropagation
+              },
+            }
+          ),
+        ])
+    end
+
+    def game_rows_data
+      @game_rows_data ||= visible_games.map do |game|
+        next if game.meta::GAME_IS_VARIANT_OF
+
+        status =
+          begin
+            parts = []
+            parts << 'Prototype' if game.meta::PROTOTYPE
+            parts << game.meta::DEV_STAGE.capitalize
+            parts.join(', ')
+          end
+
+        {
+          'title' => game.display_title,
+          'status' => status,
+          'location' => game.meta::GAME_LOCATION,
+          'meta' => game.meta,
+        }
+      end.compact
+    end
+
+    def render_game_row_title(title, game_meta)
+      style =
+        if selected_game && game_meta == (selected_game::GAME_IS_VARIANT_OF || selected_game)
+          {
+            'background-color': 'lightblue',
+          }
+        else
+          {}
+        end
+
+      render_button(title, style: style) do
+        select_game(game_meta)
+      end
+    end
+
+    def render_games_table(cols = %w[location status])
+      selected_games = game_rows_data.sort_by { |g| [g['meta']::PROTOTYPE ? 1 : 0, g['meta']] }
+      selected_games = selected_games.select { |g| %i[alpha beta production].include?(g['meta']::DEV_STAGE) } if @production
+
+      if @keywords&.size&.positive?
+        searches = @keywords.split(/[:, ]+/).map(&:upcase)
+
+        selected_games = selected_games.select do |game|
+          searches.all? do |search|
+            game['meta'].keywords.any? do |keyword|
+              keyword =~ /#{search}/
```

Comment:
would jaro winkler be better here?

Reply:
I think not; jaro winkler would assign a score to how far off each keyword is, so we'd also have to come up with some minimum score a game must reach in order to filter the list. With the current logic every game is just a match or it isn't.